diff --git a/node_modules/tsdx/dist/.progress-estimator/5e1a1bc72a1c85f144986cb7aa3d91e6fd42e1c4 b/node_modules/tsdx/dist/.progress-estimator/5e1a1bc72a1c85f144986cb7aa3d91e6fd42e1c4
index d922cf9..cdc6612 100644
--- a/node_modules/tsdx/dist/.progress-estimator/5e1a1bc72a1c85f144986cb7aa3d91e6fd42e1c4
+++ b/node_modules/tsdx/dist/.progress-estimator/5e1a1bc72a1c85f144986cb7aa3d91e6fd42e1c4
@@ -1,2 +1,10 @@
 1063
 1221
+15096
+11034
+5894
+5488
+9926
+3208
+8393
+19165
\ No newline at end of file
diff --git a/node_modules/tsdx/dist/.progress-estimator/9a3492e73be0eb3875889eb84075f018f90b4f72 b/node_modules/tsdx/dist/.progress-estimator/9a3492e73be0eb3875889eb84075f018f90b4f72
index 519236e..94cdd6d 100644
--- a/node_modules/tsdx/dist/.progress-estimator/9a3492e73be0eb3875889eb84075f018f90b4f72
+++ b/node_modules/tsdx/dist/.progress-estimator/9a3492e73be0eb3875889eb84075f018f90b4f72
@@ -1,2 +1,10 @@
 1973
 2032
+3131
+3763
+3766
+3102
+3044
+3216
+3116
+3078
\ No newline at end of file
diff --git a/node_modules/tsdx/dist/createRollupConfig.js b/node_modules/tsdx/dist/createRollupConfig.js
index d97884e..ff298b8 100644
--- a/node_modules/tsdx/dist/createRollupConfig.js
+++ b/node_modules/tsdx/dist/createRollupConfig.js
@@ -15,108 +15,105 @@ const rollup_plugin_typescript2_1 = tslib_1.__importDefault(require("rollup-plug
 const rollup_plugin_preserve_shebang_1 = tslib_1.__importDefault(require("rollup-plugin-preserve-shebang"));
 const replacements = [{ original: 'lodash', replacement: 'lodash-es' }];
 const babelOptions = {
-    exclude: /node_modules/,
-    plugins: [
-        'annotate-pure-calls',
-        'dev-expression',
-        ['transform-rename-import', { replacements }],
-    ],
+  exclude: /node_modules/,
+  plugins: [
+    'annotate-pure-calls',
+    'dev-expression',
+    ['transform-rename-import', { replacements }],
+  ],
 };
 function createRollupConfig(format, env, opts) {
-    let shebang;
-    return {
-        // Tell Rollup the entry point to the package
-        input: opts.input,
-        // Tell Rollup which packages to ignore
-        external: utils_1.external,
-        // Establish Rollup output
-        output: {
-            // Set filenames of the consumer's package
-            file: `${constants_1.paths.appDist}/${utils_1.safeVariableName(opts.name)}.${format}.${env}.js`,
-            // Pass through the file format
-            format,
-            // Do not let Rollup call Object.freeze() on namespace import objects
-            // (i.e. import * as namespaceImportObject from...) that are accessed dynamically.
-            freeze: false,
-            // Do not let Rollup add a `__esModule: true` property when generating exports for non-ESM formats.
-            esModule: false,
-            // Rollup has treeshaking by default, but we can optimize it further...
-            treeshake: {
-                // We assume reading a property of an object never has side-effects.
-                // This means tsdx WILL remove getters and setters on objects.
-                //
-                // @example
-                //
-                // const foo = {
-                //  get bar() {
-                //    console.log('effect');
-                //    return 'bar';
-                //  }
-                // }
-                //
-                // const result = foo.bar;
-                // const illegalAccess = foo.quux.tooDeep;
-                //
-                // Punchline....Don't use getters and setters
-                propertyReadSideEffects: false,
-            },
-            name: opts.name || utils_1.safeVariableName(opts.name),
-            sourcemap: true,
-            globals: { react: 'React', 'react-native': 'ReactNative' },
-            exports: 'named',
+  let shebang;
+  return {
+    // Tell Rollup the entry point to the package
+    input: opts.input,
+    // Tell Rollup which packages to ignore
+    external: utils_1.external,
+    // Establish Rollup output
+    output: {
+      // Set filenames of the consumer's package
+      file: `${constants_1.paths.appDist}/${utils_1.safeVariableName(opts.name)}.${format}.${env}.js`,
+      // Pass through the file format
+      format,
+      // Do not let Rollup call Object.freeze() on namespace import objects
+      // (i.e. import * as namespaceImportObject from...) that are accessed dynamically.
+      freeze: false,
+      // Do not let Rollup add a `__esModule: true` property when generating exports for non-ESM formats.
+      esModule: false,
+      // Rollup has treeshaking by default, but we can optimize it further...
+      treeshake: {
+        // We assume reading a property of an object never has side-effects.
+        // This means tsdx WILL remove getters and setters on objects.
+        //
+        // @example
+        //
+        // const foo = {
+        //  get bar() {
+        //    console.log('effect');
+        //    return 'bar';
+        //  }
+        // }
+        //
+        // const result = foo.bar;
+        // const illegalAccess = foo.quux.tooDeep;
+        //
+        // Punchline....Don't use getters and setters
+        propertyReadSideEffects: false,
+      },
+      name: opts.name || utils_1.safeVariableName(opts.name),
+      sourcemap: true,
+      globals: { react: 'React', 'react-native': 'ReactNative' },
+      exports: 'named',
+    },
+    plugins: [
+      rollup_plugin_node_resolve_1.default({
+        module: true,
+        jsnext: true,
+        browser: opts.target !== 'node',
+      }),
+      format === 'umd' &&
+      rollup_plugin_commonjs_1.default({
+        // use a regex to make sure to include eventual hoisted packages
+        include: /\/node_modules\//,
+      }),
+      rollup_plugin_json_1.default(),
+      rollup_plugin_typescript2_1.default({
+        typescript: require('typescript'),
+        cacheRoot: `./.rts2_cache_${format}`,
+        tsconfigDefaults: {
+          compilerOptions: {
+            sourceMap: true,
+            declaration: true,
+            jsx: 'react',
+          },
         },
-        plugins: [
-            rollup_plugin_node_resolve_1.default({
-                module: true,
-                jsnext: true,
-                browser: opts.target !== 'node',
-            }),
-            format === 'umd' &&
-                rollup_plugin_commonjs_1.default({
-                    // use a regex to make sure to include eventual hoisted packages
-                    include: /\/node_modules\//,
-                }),
-            rollup_plugin_json_1.default(),
-            rollup_plugin_typescript2_1.default({
-                typescript: require('typescript'),
-                cacheRoot: `./.rts2_cache_${format}`,
-                tsconfigDefaults: {
-                    compilerOptions: {
-                        sourceMap: true,
-                        declaration: true,
-                        jsx: 'react',
-                    },
-                },
-                tsconfigOverride: {
-                    compilerOptions: {
-                        target: 'esnext',
-                    },
-                },
-            }),
-            rollup_plugin_babel_1.default(babelOptions),
-            rollup_plugin_replace_1.default({
-                'process.env.NODE_ENV': JSON.stringify(env),
-            }),
-            rollup_plugin_sourcemaps_1.default(),
-            rollup_plugin_size_snapshot_1.sizeSnapshot({
-                printInfo: false,
-            }),
-            env === 'production' &&
-                rollup_plugin_terser_1.terser({
-                    sourcemap: true,
-                    output: { comments: false },
-                    compress: {
-                        keep_infinity: true,
-                        pure_getters: true,
-                    },
-                    ecma: 5,
-                    toplevel: format === 'es' || format === 'cjs',
-                    warnings: true,
-                }),
-            rollup_plugin_preserve_shebang_1.default({
-                shebang,
-            }),
-        ],
-    };
+        tsconfigOverride: {
+          compilerOptions: {
+            target: 'esnext',
+          },
+        },
+      }),
+      rollup_plugin_babel_1.default(babelOptions),
+      rollup_plugin_replace_1.default({
+        'process.env.NODE_ENV': JSON.stringify(env),
+      }),
+      rollup_plugin_sourcemaps_1.default(),
+      rollup_plugin_size_snapshot_1.sizeSnapshot({
+        printInfo: false,
+      }),
+      env === 'production' &&
+      rollup_plugin_terser_1.terser({
+        sourcemap: true,
+        output: { comments: false },
+        compress: false,
+        ecma: 6,
+        toplevel: format === 'es' || format === 'cjs',
+        warnings: true,
+      }),
+      rollup_plugin_preserve_shebang_1.default({
+        shebang,
+      }),
+    ],
+  };
 }
 exports.createRollupConfig = createRollupConfig;
\ No newline at end of file
